Tests/SuperValidatorTests/PhoneValidatorTests.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  File.swift
+//  
+//
+//  Created by christopher.mienarto on 23/03/22.
+//
+
+import XCTest
+
+@testable import SuperValidator
+
+internal final class PhoneValidatorTests: XCTestCase {
+    private let validator = SuperValidator.shared
+    
+    // Valid
+    internal func testValidPhoneNumberInternational() {
+        let phoneInternational = "+628916123123"
+        let isValidInternational = validator.validatePhone(phoneInternational, options: .init(phoneFormatType: .international, specifiedCountryCode: ["+62"]))
+        switch isValidInternational {
+        case .success:
+                XCTAssertTrue(true)
+                break
+            case let .failure(error):
+            XCTFail("Expected to be a success but got a failure with \(error)")
+                break
+        }
+       
+    }
+    
+    internal func testValidPhoneNumberNANP() {
+        let phoneNANP = "(415) 555 0132"
```

Comment:
what if we put 089123456789, does it valid bang ted?

Reply:
right now is invalid kak Ambuy, any suggestion on that? 
how about I give a specific validation on Indonesian Phone number, so either +62 or 0 will be accepted?